compiler/optimizer/LocalOpts.cpp

```diff
@@ -6513,8 +6513,8 @@ void TR_BlockSplitter::dumpBlockMapper(TR_LinkHeadAndTail<BlockMapper>* bMap)
          else
             traceMsg(comp(), " %d", itr->_from->getNumber());
          }
+         traceMsg(comp(), "\n");
```

Comment:
This line is indented one level too deep.
```suggestion
      traceMsg(comp(), "\n");
```